Scripts/build-phases/generate-credentials.xcfilelist

```diff
@@ -0,0 +1,12 @@
+# Input files for the script generating the app's credentials.
+# The script is currently run via a build phase in the GenerateCredentials
+# aggregate target.
+$(SRCROOT)/Credentials/replace_secrets.rb
+$(SRCROOT)/Credentials/ApiCredentials.tpl
+$(SRCROOT)/Credentials/InfoPlist.tpl
+$(SRCROOT)/Credentials/Templates/APICredentials-Template.swift
+$(SRCROOT)/Credentials/Templates/InfoPlist-Template.h
+$(HOME)/.configure/woocommerce-ios/secrets/woo_app_credentials.json
```

Comment:
The thought occurred to me to move `SECRETS_PATH` in an `.xcconfig` and use it here, so it's fully centralized. Or, at least, as centralized as it can be between the `.configure` JSON and a single place in the project.

Reply:
That seems like a nice improvement for sure 🙂